crates/aiken-lang/src/gen_uplc.rs

```diff
@@ -4071,8 +4079,8 @@ impl<'a> CodeGenerator<'a> {
                     &names_types,
                     tail,
                     term,
-                    is_expect && !tail,
                     true,
+                    expect_level,
```

Comment:
The `!tail` has been lost?

Reply:
Wasn't needed before, but now I confirmed for sure

Reply:
Well actually the proper response is I changed how the function worked to go off the tail param and the expect level. So the inner workings changed since before this pr.